### PR TITLE
feat: remaining items as required by the Biosimulations 1.0 release

### PR DESCRIPTION
**Pull request by AlexPatrie**
#### As required by the Biosimulations `1.0` release, this PR contains the following content:
- fix(ui): Fixes request logic handler in `platform/customize-simulation.component` for custom simulation archive creation calls.
- fix(style): Fixes DOM element family justifications in `platform/simulations/view` result visualization component.
- feat: Adds an auto-generated `<plot>` of all output observable results within the `<listOfOutputs>` specification for all archives created via the `simulation-project-utils/create-project` component.
- Adds `netlify.toml` specification for redirect of `dispatch -> platform` content, effectively removing clientside access to `https://run.biosimulations.org`.
- Additional adjustments as needed ahead of the `1.0` release.

### CODE REVIEW

File: apps/platform/src/app/components/run/customize-simulation/customize-simulation.component.ts [hunk 1]
```diff
@@ -600,6 +612,87 @@ export class CustomizeSimulationComponent implements OnInit, OnDestroy {
     });
   }
 
+  public _createNewArchive(queryParams: ReRunQueryParams): Promise<Blob | null> {
```
Comment (jcschaff) [inline apps/platform/src/app/components/run/customize-simulation/customize-simulation.component.ts]: @AlexPatrie is _createNewArchive() used at all?  you introduced a new method, but not a new usage of this method.

**Review by jcschaff — CHANGES_REQUESTED** [review]
please clarify

File: apps/platform/src/app/components/run/customize-simulation/customize-simulation.component.ts (hunk 1 above)
Comment (AlexPatrie) [reply apps/platform/src/app/components/run/customize-simulation/customize-simulation.component.ts]: Unused content was removed.

**Review by jcschaff — APPROVED** [review]
looks reasonable, assumed you tested it locally - approved.

**Pull request merged**